docs/0 - install.md

```diff
@@ -18,6 +18,18 @@ table to store configuration options. As such, you should run the migrations.
 
 ## Initial Setup
 
+### Command Setup
+
+1. Run the following command. This command does *Manual Setup* 1. through 3. and runs the migrations.
```

Comment:
Yours is perfectly fine but I had to read it twice to figure out the punctuation. My suggestion:
```suggestion
1. Run the following command. This command handles steps 1-3 of *Manual Setup* and runs the migrations.
```